wal/repair.go

```diff
@@ -32,15 +32,20 @@ func Repair(dirpath string) bool {
 	}
 	defer f.Close()
 
-	n := 0
+	n := int64(0)
 	rec := &walpb.Record{}
 
 	decoder := newDecoder(f)
 	for {
+		lastOff := decoder.lastOffset()
 		err := decoder.decode(rec)
 		switch err {
 		case nil:
-			n += 8 + rec.Size()
+			n += decoder.lastOffset() - lastOff
```

Comment:
we do not need this n anymore as we have lastOff at line40? n is used at line82, which can be replaced by lastOff?